wagtailtinymce/rich_text.py

```diff
@@ -78,26 +63,11 @@ def render(self, name, value, attrs=None):
         return super(TinyMCERichTextArea, self).render(name, translated_value, attrs)
 
     def render_js_init(self, id_, name, value):
-        kwargs = {
-            'options': self.kwargs.get('options', {}),
-        }
-
-        if 'buttons' in self.kwargs:
-            if self.kwargs['buttons'] is False:
-                kwargs['toolbar'] = False
-            else:
-                kwargs['toolbar'] = [
-                    ' | '.join([' '.join(groups) for groups in rows])
-                    for rows in self.kwargs['buttons']
-                ]
-
-        if 'menus' in self.kwargs:
-            if self.kwargs['menus'] is False:
-                kwargs['menubar'] = False
-            else:
-                kwargs['menubar'] = ' '.join(self.kwargs['menus'])
+        tinyMCEConfig = {}
+        if 'tinymce_config' in self.kwargs:
```

Comment:
This is weird. I don't know Python well enough to understand why it's like this. 🔢